core/embed/rust/src/ui/model_tr/component/input_methods/choice.rs

```diff
@@ -491,6 +486,13 @@ where
     }
 
     fn event(&mut self, ctx: &mut EventCtx, event: Event) -> Option<Self::Msg> {
+        // Cancel highlighting of the current choice.
+        // The Highlighting is started by pressing the middle button and
+        // canceled immediately when any other event is processed
+        if self.inverse_selected_item {
+            self.inverse_selected_item = false;
```

Comment:
nit: the if is not doing anything here

Reply:
:-) aha

Reply:
fixed here c2c346fcd2da325d94509d6514e6dc0f0793d283